documents/invoice.md

```diff
@@ -239,7 +253,7 @@ data:= map[string]interface{}{
     "updated-key": "An updated note.",
   },
 }
-body, err := client.Invoice.Invoice.Update("<invoiceId>", data, nil)
+body, err := client.Invoice.Update("<invoiceId>", data, nil)
 
```

Comment:
returns NULL

Reply:
> returns NULL

can you try with this once 

`	line_items := make(map[string]interface{})
	line_items["0"] = map[string]interface{}{
		  "id": "li_JpNz5IZZcAtRFU",
		  "name": "Book / English August - Updated name and quantity",
		  "quantity": 1,
		}
	
	data:= map[string]interface{}{
	  "line_items": line_items,
	  "notes": map[string]interface{}{
		"updated-key": "An updated note.",
	  },
	}`

	`body, err := client.Invoice.Update("inv_JpNz5HmNXYvvNL", data, nil)`